list.go

```diff
@@ -50,3 +50,13 @@ type ListLookUpByIDResponse struct {
 	Detail   string              `json:"detail,omitempty"`
 	Type     string              `json:"type,omitempty"`
 }
+
+type ListsTweetsLookUpByIDResponse struct {
+	Tweets   []*Tweet            `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
+	Errors   []*APIResponseError `json:"errors,omitempty"`
+	Meta     *ListMeta           `json:"meta"`
+	Title    string              `json:"title,omitempty"`
```

Comment:
Run this command and then invalid request error return this struct.
`curl "https://api.twitter.com/2/lists/1111111111111111111111111111111111111111/tweets" -H "Authorization: Bearer $token"`